### DISCUSSION TITLE
Hello, is something just change today on the sdk https://sdk.mercadopago.com/js/v2 ?

### DISCUSSION THREAD

**Discussion by emilioHayes** [mercadopago/sdk-js#152] [Q&A] [answered]
Hello everyone, i have a website where i implement from 2 years ago the php sdk for server side and the checkout API, where i use the cardform, in my website i only display the necesary fields:
Email
Name Holder
Card Number
Expiration date
CVC

And from 2 years ago everything works good, but just today with no changes on the website, the js sdk returns me the next error:
error: "bad_request"
message: "Invalid cardholder.identification.type: CURP in site_id: MLM"
status: 400

So that field i never displayed so i never fill it and it was working good from almost two years until today. Even if i fill this field correctly it still throgin that error. Weird. 
So i guess something changes... does somebody knows about it ?

Im thinking in re-built the integration, because i dont remember too much about it to try to fix it.

Please Help

**Comment by AdaltonLeite** [comment]
Hi @emilioHayes, 

Recently we have not had any updates to our JS SDK that generate this kind of issue. We did some tests and could not simulate the problem, does the error still persist?

**Comment by emilioHayes** [comment, marked as answer]
Hello! Thank you for your reply!
The error is gone now! Thank god!
The error was there for almost the whole day yesterday, at midnight the
error was gone!

I hope this dont happen again.
